### DISCUSSION TITLE
Is it possible to put my slinger source on webpage?

### DISCUSSION THREAD

**Discussion by glaypeng78** [GerryDazoo/Slinger#630] [Q&A]
I was wondering whether it's possible to put my slinger source on the webpage so that it can be viewed in pip mode on iphone. I have already try video.js in hls and flv.js in http-flv. However unfortunately none of them work. I even try to re-stream it to a self-host nginx rtmp server but also not works. Is there any way to modify the python code to make it stream on webpage or can I have the format of slinger to encode. So that I can find the proper way to stream on webpage.

**Comment by flipper2222222** [comment]
an alternative for you is an approach that user rbf121 helped me with. send slingbox to plex and the plex client on the phone can run in PIP mode, if you are interested in finding out more, let me know and I will look up how it was done. note there is more latency with this type of configuration.

**Reply by BRGit8** [reply]
I use an app called Outplayer on my iphone. It works great and has PIP. I hope it helps.

**Reply by glaypeng78** [reply]
@flipper2222222 thanks for your solution. However after trying @BRGit8 mentioned "Outplayer". It 100% fit my need. With this app on my iphone I was able to stream slinger source in pip and press button of controller in webpage at the same time. And I think with directly stream the source it may have less latency comparing with re-stream to PLEX. Once again thank both of your replies.

**Reply by flipper2222222** [reply]
outplayer doesn't have a PIP button for me, how are you guys getting it into PIP mode?

**Reply by BRGit8** [reply]
Just change to another app and leave outplayer open. It will appear in PIP. And you can move and resize it.

**Reply by flipper2222222** [reply]
thanks for the tip, 
have you found a way to fill the screen in landscape mode?
also
a way to save URL in the app?

cheers

**Reply by BRGit8** [reply]
When I turn the phone to landscape position, the picure fill the screen. I forgot to mention that when you double click the Pip, it resizes it. There's no way to save the URL. I contacted the developer and asked for this on next upgrade. Let's see.

**Reply by glaypeng78** [reply]
Yep, it automatically turns into landscape when rotate your phone to horizontal. So far I save my domain name for slingbox in VLC player and copy to Outplayer. However I also looking forward to have the bookmark in next update.

**Reply by flipper2222222** [reply]
I also contacted the  developer about saving URLs_

I have 30mm of black space at each side in landscape mode Plex has 15mm.  The iplayer  Extreme application allows stretch to fill screen  but no PIP mode yet. The developers say "they are working on it"

**Comment by flipper2222222** [comment]
outplayer sounds like a great solution ,  I am going to try today , thanks a lot
